draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3923,14 +3982,13 @@ RESPONSE:
             "trigger-subject": "content",
             "generic-trigger-spec-type": "content-objectlist",
             "generic-trigger-spec-value": {
-              "href":
-                "https://www.example.com/hls/title/index.m3u8",
-              "type": "hls"
-            }
+              "objects": [{
+                "href": "https://www.example.com/hls/title/index.m3u8",
```

Comment:
This line is now triggering an artwork too wide warning in idnits.  We need to find some way to shorten it by a couple of characters.